### PR TITLE
add some more information about hashbangctl

### PR DESCRIPTION
**Pull request by napcae**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,20 @@ User config tool for #!
...
-TODO. Document
+## Documentation 
 
-Install: `pip install git+https://github.com/hashbang/hashbangctl`
+Typing ``hashbangctl`` when you're connected to one of the hashbang servers will start the programm and give you some details about your user.
+Follow the instructions on the screen to change your 
+
+* username 
+* shell(``cat /etc/shells`` will list all available shells) 
+	* add or delete SSH Keys
```
Comment (daurnimator) [inline README.md]: indentation here isn't consistent: it's okay to use tabs, but you still need to use a space before the `*`.
Note that usually we use `  - ` (`<space><space><hyphen><space>`) to create markdown lists. So the indentend form with tabs is: `<tab><space><space><hyphen><space>`

File: README.md [hunk 2]
```diff
@@ -2,8 +2,20 @@ User config tool for #!
...
+	* import SSH Keys from github
+
+Remember to save your changes, quitting the programm will discard your changes.
```
Comment (daurnimator) [inline README.md]: typo: programm

File: README.md [hunk 3]
```diff
@@ -2,8 +2,20 @@ User config tool for #!
...
+
+
+## Installation
```
Comment (daurnimator) [inline README.md]: new line after heading.

File: README.md [hunk 4]
```diff
@@ -2,8 +2,21 @@ User config tool for #!
...
 
-TODO. Document
+## Documentation 
 
-Install: `pip install git+https://github.com/hashbang/hashbangctl`
+Typing ``hashbangctl`` when you're connected to one of the hashbang servers will start the programm and give you some details about your user.
+Follow the instructions on the screen to change your 
+
+- username 
+- shell(``cat /etc/shells`` will list all available shells)
```
Comment (daurnimator) [inline README.md]: Use single backticks for code sections; not double.

File: README.md [hunk 5]
```diff
@@ -2,8 +2,21 @@ User config tool for #!
 
 Lets users change ldap details, ssh keys, etc.
 
-TODO. Document
```
Comment (daurnimator) [inline README.md:5]: Need two new lines at end of section

File: README.md [hunk 6]
```diff
@@ -2,8 +2,21 @@ User config tool for #!
...
+## Documentation 
 
-Install: `pip install git+https://github.com/hashbang/hashbangctl`
+Typing ``hashbangctl`` when you're connected to one of the hashbang servers will start the programm and give you some details about your user.
+Follow the instructions on the screen to change your 
+
+- username
```
Comment (daurnimator) [inline README.md]: These two dot points are differently indented than the SSH ones below

**Review by daurnimator — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (napcae) [reply README.md]: it's intentional

Comment (napcae) [reply README.md]: either you change your (1)username, (2)shell or change information about your (3)ssh keys

Comment (daurnimator) [reply README.md]: Then you need a top level list element for the two others to be a child of

File: README.md [hunk 7]
```diff
@@ -2,8 +2,22 @@ User config tool for #!
...
-TODO. Document
 
-Install: `pip install git+https://github.com/hashbang/hashbangctl`
+## Documentation 
+
+Typing ``hashbangctl`` when you're connected to one of the hashbang servers will start the programm and give you some details about your user.
+Follow the instructions on the screen to change your 
+
+- username 
+- shell(`cat /etc/shells` will list all available shells)
```
Comment (daurnimator) [inline README.md:12]: space before parenthesis?

File: README.md [hunk 8]
```diff
@@ -2,8 +2,22 @@ User config tool for #!
 
 Lets users change ldap details, ssh keys, etc.
 
-TODO. Document
 
-Install: `pip install git+https://github.com/hashbang/hashbangctl`
+## Documentation 
+
+Typing ``hashbangctl`` when you're connected to one of the hashbang servers will start the programm and give you some details about your user.
```
Comment (daurnimator) [inline README.md:8]: typo: programm

**Comment by KellerFuchs** [comment]
@napcae What's up?  This should be mergeable, assuming you address @daurnimator's comments.

**Comment by KellerFuchs** [comment]
@daurnimator TBH, I kinda feel like this is the sort of stuff for which pushing a fix to the PR branch directly is OK.  Having a bunch of extra round-trips mostly feels tiresome  :3

**Pull request closed without merging**